packages/docs/src/styleguide.md

```diff
@@ -19,7 +19,7 @@ npm install --save @blueprintjs/core react react-dom react-addons-css-transition
 Import components from the `@blueprintjs/core` module into your project.
 Don't forget to include the main CSS stylesheet too!
 
-**[See Components Usage for more complete installation instructions.](#components.usage)**
+[See **Components usage** for more complete installation instructions.](#components.usage)
```

Comment:
Fair, I'm gonna match what we do in all other files:
`Make sure to review the [general usage docs for JS components](#components.usage).`